h2o-genmodel/src/main/java/hex/genmodel/algos/tree/ConvertTreeOptions.java

```diff
@@ -0,0 +1,23 @@
+package hex.genmodel.algos.tree;
+
+public class ConvertTreeOptions {
+  static final ConvertTreeOptions DEFAULT = new ConvertTreeOptions();
+
+  boolean _checkTreeConsistency = false;
+
+  /**
+   * Performs a self-check on each converted tree. Inconsistencies are reported in the log.
+   * @return self
+   */
+  public ConvertTreeOptions enableCheckTreeConsistency() {
```

Comment:
if we make this immutable we can keep this method, only it would return a new object and would probably need a different name like `withTreeConsitencyCheckEnabled()`